src/showcase/effects/BackgroundGrid.tsx

```diff
@@ -1,18 +1,23 @@
+import { HTMLProps } from 'react'
+
+interface BackgroundGridProps {
+  color: string
+  cellSize: string | number
+  strokeWidth: number | string
+  className?: string
+  fade?: boolean
+}
+
 const BackgroundGrid = ({
-  color = 'black',
-  size = '35px',
-  strokeWidth = '1px',
+  color = '#a1a1a1',
+  cellSize = '25px',
+  strokeWidth = '3px',
   className,
   fade = true,
-}: {
-  color?: string
-  size?: string | number
-  strokeWidth?: number | string
-  className?: string
-  fade?: boolean
-}) => {
+  ...props
+}: Partial<BackgroundGridProps> & HTMLProps<HTMLDivElement>) => {
   const svg = `
-    <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 200 200' stroke='${color}' stroke-width='${strokeWidth}'>
+    <svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 200 200' stroke='${color}' stroke-width='${strokeWidth}' fill-opacity='0.4' >
```

Comment:
make opacity full but change color hex code to change the opacity. 

for eg. `#a1a1a140`

Reply:
Do you want me to change this in the other backgrounds too?

Reply:
I put the 0.4 in because that is in the other components. Which should we use: `#a1a1a140` or `#a1a1a1`?

Reply:
`#a1a1a140` looked too light so i used `#a1a1a1`

Reply:
i see cool cool let's leave it then